tests/foreman/api/test_provisioning.py

```diff
@@ -147,3 +148,123 @@ def test_rhel_pxe_provisioning(
     # assert that the host is subscribed and consumes
     # subsctiption provided by the activation key
     assert provisioning_host.subscribed, 'Host is not subscribed'
+
+
+@pytest.mark.e2e
+@pytest.mark.parametrize('pxe_loader', ['ipxe'], indirect=True)
+@pytest.mark.on_premises_provisioning
+@pytest.mark.rhel_ver_match('[8]')
+def test_rhel_ipxe_provisioning(
+    request,
+    module_provisioning_sat,
+    module_sca_manifest_org,
+    module_location,
+    provisioning_host,
+    pxe_loader,
+    module_provisioning_rhel_content,
+    provisioning_hostgroup,
+    module_lce_library,
+    module_default_org_view,
+):
+    """Provision a host using iPXE workflow
+
+    :id: 9e016e1d-757a-48e7-9159-131bb65dc4ed
+
+    :steps:
+        1. Configure satellite for provisioning
+        2. provision a host
+        3. Check that resulting host is registered to Satellite
+        4. Check host is subscribed to Satellite
+
+    :expectedresults:
+        1. Provisioning via iPXE is successful
+        2. Host installs right version of RHEL
+        3. Satellite is able to run REX job on the host
+        4. Host is registered to Satellite and subscription status is 'Success'
+
+    :parametrized: yes
+    """
+    # TODO: parametrize iPXE Chain BIOS as pxe loader after #BZ:2171172 is fixed
+    sat = module_provisioning_sat.sat
+    # set http url
+    ipxe_http_url = sat.install(
```

Comment:
Do you mean `sat.execute`?

Reply:
No, sat.execute doesn't work here . I get `TypeError: Expected bytes, got InstallerCommand`